src/site/stages/build/plugins/download-drupal-assets.js

```diff
@@ -16,6 +16,9 @@ async function downloadFile(
   everythingDownloaded,
 ) {
   const asset = assetsToDownload.shift();
+  if (!asset) {
+    return;
+  }
```

Comment:
Apparently, undefined objects can end up in `assetsToDownload` somehow?

Reply:
No idea, how did you find this?

Reply:
My build broke, and this fixed it.